.github/ISSUE_TEMPLATE.md

```diff
@@ -5,6 +5,7 @@ While not necessary, you can help organize our issues by labeling this issue whe
 
 - [ ] has-reproduction
 - [ ] feature
+- [ ] docs
 - [ ] blocking
 - [ ] good first issue
```

Comment:
Should this `good first issue` also be removed?

Reply:
I think it's reasonable to have the `good first issue`, since I believe(possibly naively) that people will be able to tell determine a good issue for a first timer, such as docs, etc. Maintainers probably will not look at the `good first issue` label, so if it generates too much noise we can remove it. My hunch is that there is huge value in having some issues labelled as good first issue, so that someone has a place to start, rather than being paralyzed by a list of random labels.

I've also seen other repositories put a lot of effort into an issue to make it a good first one, so I'm hopeful that that will translate to the opener doing something similar